lib/src/domain/entity/movie_category.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:floor/floor.dart';
+import 'package:flutter/foundation.dart';
+
+import '../../core/util/enums.dart';
+import 'movie.dart';
+
+@Entity(
+  primaryKeys: <String>['movieId', 'category'],
+  foreignKeys: <ForeignKey>[
+    ForeignKey(
+      childColumns: <String>['movieId'],
+      parentColumns: <String>['id'],
+      entity: Movie,
+    ),
+    ForeignKey(
+      childColumns: <String>['category'],
+      parentColumns: <String>['category'],
+      entity: Category,
+    ),
+  ],
+)
+class MovieCategory {
+  final Endpoint category;
+  final int movieId;
+
+  MovieCategory({
+    required this.category,
+    required this.movieId,
+  });
+}
```

Comment:
This class is duplicated? You have the same definition on `lib/src/domain/entity/category.dart`

Reply:
What, strange. In my Android Studio i see two diferent classes.
![image](https://github.com/MateoManolio/movie_card/assets/84589343/aa69f357-6655-4ab1-b9cc-916bc5cf744a)
![image](https://github.com/MateoManolio/movie_card/assets/84589343/43e3b9c3-2e53-4840-8501-4fe003b4183c)